client/src/Components/CoffeeShop.js

```diff
@@ -2,11 +2,13 @@ import React, { useState, useEffect, useContext } from "react";
 import axios from "axios";
 import ReviewForm from "./ReviewForm";
 import CoffeeShopReview from "./CoffeeShopReview";
+import CoffeeShopForm from "./CoffeeShopForm";
 
 const CoffeeShop = ({ match, history }) => {
   const [shop, setShop] = useState(null);
   const [reviews, setReviews] = useState([]);
   const [showForm, setShowForm] = useState(false);
+  const [showEditForm, setShowEditForm] = useState(false);
```

Comment:
Change "showForm" to "showReviewForm" 
and "showEditForm" to "showShopEditForm"